ProjFS.Mac/PrjFSLib.Mac.Managed/VirtualizationInstance.cs

```diff
@@ -193,8 +194,7 @@ public virtual Result StopVirtualizationInstance()
                     return this.OnPreDelete(relativePath, isDirectory);
 
                 case NotificationType.FileModified:
-                    this.OnFileModified(relativePath);
-                    return Result.Success;
+                    return this.OnFileModified(relativePath, isPlaceholderFile);
```

Comment:
`OnFileModified` should stay as `void` return type.  Unlike `OnPreDelete` or `OnFilePreConvertToFull` this callback is informational.  If the provider returns a failure code we can't undo the modification, and the native library should still convert the file to full (i.e. remove the placeholder attributes).